toqito/channels/reduction.py

```diff
@@ -37,10 +37,16 @@ def reduction(dim: int, k: int = 1) -> np.ndarray:
      [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  1.,  0.],
      [-1.,  0.,  0.,  0., -1.,  0.,  0.,  0.,  0.]])
 
+
+    References
+    ==========
+    .. bibliography::
+        :filter: docname in docnames
+
     :param dim: A positive integer (the dimension of the reduction map).
     :param k: If this positive integer is provided, the script will instead return the Choi
               matrix of the following linear map: Phi(X) := K * Tr(X)I - X.
     :return: The reduction map.
     """
     psi = max_entangled(dim, False, False)
-    return k * identity(dim**2) - psi * psi.conj().T
+    return k * identity(dim ** 2) - psi * psi.conj().T
```

Comment:
Actually, if you've already made a change to this line, I believe the more proper way to do multiplication is by way of the `@` symbol. When I started `toqito`, this wasn't the case, but I've been trying to replace `*` with `@`where appropriate (i.e. anywhere there is vector-to-vector or vector-to-matrix multiplication).

```suggestion
return k * identity(dim ** 2) - psi @ psi.conj().T
```